android/start.md

```diff
@@ -100,19 +109,30 @@ b. To generate the Java classes for these models, click the Gradle Task dropdown
 
 ## Step 3: Add API and Database
 
-a. Run `amplify configure` from the root of your application folder to set up Amplify with your AWS account.
+a. Run `amplify configure` in Terminal from the root of your application folder to set up Amplify with your AWS account.
+
+    - Browser will pop up prompting you to sign in / create new account
+    - Once done, return to terminal and press Enter
+    - Choose a region
+    - Choose a username (can use default)
+    - Browser will popup again to finish creating that user - you can use all default properties.
```

Comment:
Your default browser will open a tab prompting you to walkthrough the process of creating an IAM user. At the end of the process. Save the Access ID and Secret key